src/AzureFunctionPackage/Functions/Common/config.xml

```diff
@@ -54,8 +54,10 @@
     <merge from="release/2.1.2xx" to="release/2.1.3xx" />
     <merge from="release/2.1.3xx" to="release/2.1.4xx" />
     <merge from="release/2.1.4xx" to="release/2.1.5xx" />
+    <merge from="release/2.1.5xx" to="release/2.1.6xx" />
     <merge from="release/2.1.5xx" to="release/2.2.1xx" />
     <merge from="release/2.2.1xx" to="release/2.2.2xx" />
+    <merge from="release/2.1.6xx" to="release/2.2.2xx" />
```

Comment:
Why do changes to 2.1.6xx skip 2.2.1xx?

Wait, I think I know, but it's confusing and maybe problematic.

2.1.6xx will have Dev16 tooling components + 2.1 runtime
2.2.1xx will have Dev15 tooling components + 2.2 runtime

This means that you could have a project that builds with 2.1.6xx, but not 2.2.1xx because it uses a Dev16 tooling feature.

I don't think our versioning scheme accounted for non-increasing feature set possibility. :(